tests/tck/features/user/User.feature

```diff
@@ -280,3 +280,85 @@ Feature: User & privilege Test
       DROP SPACE user_tmp_space_3;
       """
     Then the execution should be successful
+
+  Scenario: Describe User
+    When executing query:
+      """
+      CREATE SPACE IF NOT EXISTS user_tmp_space_4(partition_num=1, replica_factor=1, vid_type=FIXED_STRING(8))
+      """
+    Then the execution should be successful
+    And wait 10 seconds
+    When executing query:
+      """
+      CREATE USER IF NOT EXISTS user1 WITH PASSWORD "pwd1"
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      GRANT ROLE ADMIN ON user_tmp_space_4 TO user1
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      CREATE USER IF NOT EXISTS user2 WITH PASSWORD "pwd2"
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      GRANT ROLE ADMIN ON user_tmp_space_4 TO user2
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      SHOW USERS
+      """
+    Then the result should contain:
+      | Account |
+      | "root"  |
+      | "user1" |
+      | "user2" |
+    When executing query:
+      """
+      DESC USER root
+      """
+    Then the result should be, in any order, with relax comparison:
+      | Account | Roles in spaces     |
+      | "root"  | " GOD in ALL_SPACE" |
+    When executing query:
+      """
+      DESC USER user1
+      """
+    Then the result should be, in any order, with relax comparison:
+      | Account | Roles in spaces              |
+      | "user1" | " ADMIN in user_tmp_space_4" |
```

Comment:
Maybe following table format is better:

 | Role | Space |
|-----|----|
 | ADMIN | tmp_space_4 |
| USER | tmp_space_2 |
| GUEST | space3 |

They can use pipe to further filter what they want. WDYT? For example:

```sql
desc user user1 | YIELD $-.role where $-.space == "space3" 
```

In addition, do we have `DESC USERS` commands to show all users for root? if exists, its output table maybe:

 | USER | Role | Space |
|-----|-----|----|
| user1 | ADMIN | tmp_space_4 |
| user1 | USER | tmp_space_2 |
| user2 | GUEST | space3 |